docs/managing-dragonfly/operator/prometheus-guide.md

```diff
@@ -0,0 +1,106 @@
+---
+sidebar_position: 4
+---
+
+# Integrate Prometheus with Dragonfly operator
+
+This guide provides a step-by-step instruction to set up Prometheus with Dragonfly
```

Comment:
This guide provides step-by-step instructions to set up Prometheus with the Dragonfly operator. In this guide, you'll use the Prometheus Operator and PodMonitor to manage Prometheus resources.